src/_playground/App.scss

```diff
@@ -73,6 +73,17 @@ $fd-fonts-path: '../../node_modules/fiori-fundamentals/dist/fonts/';
             overflow-y: scroll;
             background-color: white;
             padding: 2rem;
+          
+            .frDocs-Content__githubLogo {
+                float: right;
+                z-index: 100;
+                cursor: pointer;
+            }
+
+            .frDocs-Content__githubLogo svg {
+                width: 2rem;
+                height: 2rem;
+            }
```

Comment:
You can write this cleaner since it's a Sass file.
```scss
.frDocs-Content__githubLogo {
    float: right;
    z-index: 100;
    cursor: pointer;

    svg {
        width: 2rem;
        height: 2rem;
    }
}
```